system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -167,6 +167,25 @@
                 :concept-id supplied-concept-id}
                (ingest/parse-ingest-body :xml response)))))))
 
+(deftest subscription-ingest-with-bad-query-error-test
+  (testing "ingest of a new subscription concept"
+    (let [coll1 (data-core/ingest-umm-spec-collection
+                  "PROV1"
+                  (data-umm-c/collection
+                   {:ShortName "coll1"
+                    :EntryTitle "entry-title1"})
+                  {:token "mock-echo-system-token"})
+          metadata {:CollectionConceptId (:concept-id coll1)
+                    :Query "options%5Bspatial%5D%5Bor%5D=true"}
+          concept (subscription-util/make-subscription-concept metadata)
+          response (ingest/ingest-concept
+                    concept
+                    {:accept-format :json
+                     :raw? true})
+          {:keys [errors]} (ingest/parse-ingest-body :json response)]
+      (is (re-find #"Error ingesting subscription: invalid query parameters" (first errors))))))
```

Comment:
We should check the error message has the specific error message search-app returned about the invalid query parameter.